formatter/index.js

```diff
@@ -3,4 +3,22 @@ module.exports = {
   tabWidth: 2,
   singleQuote: true,
   semi: true,
-};
+  printWidth: 120,
```

Comment:
Hier bin ich irgendwie zwiegespalten. Manchmal hätte ich in der IDE gerne weniger Umbrüche. Allerdings kann man sich bei max. 80 Zeichen pro Zeile auf ungefähr "jedem" Bildschirm einen Side-by-Side-Diff ohne horizontale Scrollbalken bzw. Umbrüche anschauen (ich sehe hier in GitHub in der Split-Ansicht z. B. nur ca. 92 Zeichen pro Side, wenn der File-Tree ausgeklappt ist).

Reply:
Wir können es auch gerne auf 80 Zeichen festlegen.

Reply:
Ich wäre dafür. Wie sieht der Rest das?

Reply:
Gerne alle Änderungswünsche und Anregungen her!

Reply:
ich finde weniger zeichen besser, also eher 80 als 120